changelog/@unreleased/pr-487.v2.yml

```diff
@@ -0,0 +1,9 @@
+type: fix
+fix:
+  description: |-
+    Use Base64 URL decoder
+
+    ## Before this PR
```

Comment:
Did you mean to commit this to the changelog?

Reply:
Nope, fixed